xml/security_cryptofs.xml

```diff
@@ -264,9 +273,150 @@
   </para>
   <para>
     Keep in mind that the encrypted file can be only decrypted using the same
-    key that was used for encryption. If you want to share an encrypted file
+    key that was used for encryption. To share an encrypted file
     with another person, you have to use that person's public key to encrypt
     the file.
   </para>
  </sect1>
+ <sect1 xml:id="sec-security-cryptofs-rage">
+  <title>Encrypting files with Rage</title>
+  <para>
+   Rage is a secure file encryption software to encrypt files. It has keys that
+   are easy to exchange with other people, and has secure defaults to prevent
+   accidental misuse or leaks of sensitive data. We recommend Rage to encrypt
+   files.
+  </para>
+  <para>
+   The recipient must first generate a key pair to encrypt a file with Rage:
```

Comment:
You need to say sender/recipient because else the *sender* may think they need to generate the keys. Previously the wording was confusing and by not distinguishing made it very confusing.

Reply:
I agree with retaining recipient( Since I ve taken the maintainer role for the Security/Hardening Book , I will go going through in-depth and looking at the existing content incl GPG).